src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PhysicalFileProvider.cs

```diff
@@ -161,7 +161,13 @@ internal PhysicalFilesWatcher CreateFileWatcher()
             string root = PathUtils.EnsureTrailingSlash(Path.GetFullPath(Root));
```

Comment:
Why this method has no annotation?

Reply:
Same reason as https://github.com/dotnet/runtime/pull/56189#discussion_r675238380.